tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -70,17 +68,17 @@ jobs:
 
   - script: |
       set -e -x
-      cd android_test/android
-      appcenter test run espresso \
-        --app "AI-Frameworks/ORT-Mobile-Android" \
-        --devices $(app_center_android_test_devices) \
-        --app-path ./app/build/outputs/apk/debug/app-debug.apk \
-        --test-series "master" \
-        --locale "en_US" \
-        --build-dir ./app/build/outputs/apk/androidTest/debug \
-        --token $(app_center_api_token)
-    displayName: Run E2E tests using App Center
-    workingDirectory: $(Build.BinariesDirectory)
+      cd tools/python
+      pip install requests
+      python upload_and_run_browserstack_tests.py \
+        --test_platform espresso \
+        --id $(browserstack_username) \
+        --token $(browserstack_access_key) \
```

Comment:
Is there WIP documentation somewhere that provides the high level details of how this all works and where these values are defined so that we know how to update them in the future?

Reply:
Yes I'll do my best to document everything in the onenote!